components/kyma-environment-broker/internal/orchestration/manager/manager.go

```diff
@@ -88,8 +107,21 @@ func (m *orchestrationManager) resolveOperations(o *internal.Orchestration) ([]o
 		for _, r := range runtimes {
 			windowBegin := time.Time{}
 			windowEnd := time.Time{}
+
+			maintenanceDays := r.MaintenanceDays
+			maintenanceWindowBegin := r.MaintenanceWindowBegin
+			maintenanceWindowEnd := r.MaintenanceWindowEnd
+
+			for _, p := range policies {
+				if p.GlobalAccountID == r.GlobalAccountID && p.Plan == r.Plan { // TODO: with what should I compare p.Plan?
```

Comment:
You should separate the matching of GA and Plan. Both of these match attributes are optional, but if one is specified, and if it's not matching the runtime's attribue, we should skip the rule entry. I think for GA we should support regexp instead of exact match only. 

So something like:
```
if p.Match.Plan != "" && p.Match.Plan != r.Plan {
	continue
}
if p.Match.GlobalAccountID != "" {
	matched, err := regexp.MatchString(p.Match.GlobalAccountID, r.GlobalAccountID)
	if err != nil || !matched {
		continue
	}
}

// We have a rule match here, either be one or all of the rule match options. Let's override maintenance attributes.
if len(p.Days) > 0 {
	maintenanceDays = p.Days
}
if !p.TimeBegin.IsZero() {
	maintenanceWindowBegin = p.TimeBegin
}
if !p.TimeEnd.IsZero() {
	maintenanceWindowEnd = p.TimeEnd
}
```